gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -7,17 +7,37 @@
 module PostgresHaAdmin
   class FailoverMonitor
     FAILOVER_ATTEMPTS = 10
-    DB_CONNECTED_CHECK_FREQUENCY = 300
+    DB_CHECK_FREQUENCY = 300
     FAILOVER_CHECK_FREQUENCY = 60
+    attr_accessor :failover_attempts, :db_check_frequency,
+                  :failover_check_frequency
 
     def initialize(db_yml_file = '/var/www/miq/vmdb/config/database.yml',
                    failover_yml_file = '/var/www/miq/vmdb/config/failover_databases.yml',
+                   ha_admin_yml_file = '/var/www/miq/vmdb/config/ha_admin.yml',
                    log_file = '/var/www/miq/vmdb/log/ha_admin.log',
                    environment = 'production')
       @logger = Logger.new(log_file)
       @logger.level = Logger::INFO
       @database_yml = DatabaseYml.new(db_yml_file, environment)
       @failover_db = FailoverDatabases.new(failover_yml_file, @logger)
+      begin
+        ha_admin_yml = YAML.load_file(ha_admin_yml_file)
+      rescue
+        ha_admin_yml = {}
+        @logger.info("File not found #{ha_admin_yml_file}. "\
+                     "Default settings for failover will be used.")
+      end
+
+      @failover_attempts = ha_admin_yml['failover_attempts'] \
+                           || FAILOVER_ATTEMPTS
```

Comment:
Please don't use line continuation...it's much cleaner to just keep the line long.